### ISSUE TITLE
NSPanel shows only the time after moving device to another instance of HA

### ISSUE THREAD

**Issue by Pheerink** [Blackymas/NSPanel_HA_Blueprint#1738]
### TFT Version

V 4.2.4

### ESPHome Version

V 4.2.4

### Blueprint Version

V 4.2.4

### Panel Model

NSPanel EU Model

### What is the bug?

The display shows only the time after moving NSPanel from one instance to another instance

### Steps to Reproduce

 I have moved a working NSpanel from DEV to PROD and since then I havea problem. The panel shows only the time instead of the different pages. Tried everything to get it right. Reinstalled several times the  ESPhome, TFT, Blueprint etc. Nothing works. The NSpanle shows only the time. It is however controllable from the Device panel. I am able to operate the relays etc.  

Updated the software TFT, and blueprint but the error remains. It shows  "blueprint pending"  in the log and I don't see the version number of the blueprint at startup. The other version numbers are shown.

### ESPHome logs

```logs
[D][sensor:094]: 'mynewnspanel RSSI': Sending state -61.00000 dBm with 0 decimals of accuracy
[21:15:26][D][sensor:094]: 'ntc_source': Sending state 1.56400 V with 2 decimals of accuracy
[21:15:26][D][resistance:039]: 'resistance_sensor' - Resistance 10090.3Ω
[21:15:26][D][sensor:094]: 'resistance_sensor': Sending state 10090.32324 Ω with 1 decimals of accuracy
[21:15:26][D][ntc:026]: 'mynewnspanel Temperature' - Temperature: 24.8°C
[21:15:26][D][sensor:094]: 'mynewnspanel Temperature': Sending state 24.79778 °C with 1 decimals of accuracy
[21:15:30][I][script.watchdog:3115]: Wi-Fi: Fair (-61 dBm)
[21:15:30][I][script.watchdog:3135]: API: Connected

[21:15:30][W][script.watchdog:3155]: Blueprint: Pending

[21:15:30][I][script.watchdog:3156]: Requesting blueprint settings
[21:15:30][I][script.watchdog:3166]: ESPHome:
[21:15:30][I][script.watchdog:3167]: Version: 4.2.4
[21:15:30][I][script.watchdog:3172]: Framework: esp-idf
[21:15:30][I][script.watchdog:3176]: UART:
[21:15:30][I][script.watchdog:3177]: Baud rate: 115200 bps
[21:15:30][I][script.watchdog:3178]: Queue size: 8
[21:15:30][I][script.watchdog:3182]: Nextion:
[21:15:30][I][script.watchdog:3183]: Queue size: 2
[21:15:30][I][script.watchdog:3185]: Is setup: True
[21:15:30][I][script.watchdog:3192]: Init: True
[21:15:30][I][script.watchdog:3200]: TFT: 4.2.4
[21:16:04][D][sensor:094]: 'mynewnspanel RSSI': Sending state -60.00000 dBm with 0 decimals of accuracy
```

### Home Assistant logs

```logs
Logger: homeassistant.components.automation.nspanel
Source: helpers/script.py:1792
Integration: Automation (documentation, issues)
First occurred: January 23, 2024 at 4:26:34 PM (13336 occurrences)
Last logged: 9:08:31 PM

NSPanel : Error executing script. Service not found for choose at pos 2: Service esphome.nspanel_in_werkkamer_send_command_text_printf not found.
NSPanel : Main choices: NSPanel event: Choose at step 2: Boot init: If at step 1: Error executing script. Service not found for call_service at pos 5: Service esphome.nspanel_in_werkkamer_page_home not found.
NSPanel : Main choices: NSPanel event: Choose at step 2: Boot init: Error executing script. Service not found for if at pos 1: Service esphome.nspanel_in_werkkamer_page_home not found.
NSPanel : Main choices: NSPanel event: Error executing script. Service not found for choose at pos 2: Service esphome.nspanel_in_werkkamer_page_home not found.
NSPanel : Error executing script. Service not found for choose at pos 2: Service esphome.nspanel_in_werkkamer_page_home not found.
```

**Comment by edwardtfn** [comment]
Could you please share the device's yaml settings?
Please hide any sensitive info before sharing.

The error looks to be related to a naming mismatch. This typically happens when you change the panel's entities names after integrating it to Home Assistant, but could also be related to the use of friendly name in the panel's yaml, etc.
It could also be related to an older version of ESPHome component, so please double check the versions shown on the blue boot screen when your panel is starting.
My suggestion would be to "Reconnect the Panel's device to Home Assistant". Please take a look at this troubleshooting guide related to panel not connecting to the Blueprint as the instructions are there: https://github.com/Blackymas/NSPanel_HA_Blueprint/blob/main/docs/error_initializing.md#blueprint-is-not-detected

Please let me know your results.

**Comment by Pheerink** [comment]
That what you suggested in the email, has been tried several times. With the install of the blueprint the installation gets into a loop and complains that the blueprint version is wrong and more often it displays "pending". Please see the following installation lines:


1.74500 V with 2 decimals of accuracy [09:55:46][D][resistance:039]: 'resistance_sensor' - Resistance 12568.5Ω [09:55:46][D][sensor:094]: 'resistance_sensor': Sending state 12568.48926 Ω with 1 decimals of accuracy [09:55:46][D][ntc:026]: 'neotest Temperature' - Temperature: 19.9°C [09:55:46][D][sensor:094]: 'neotest Temperature': Sending state 19.94252 °C with 1 decimals of accuracy [09:55:47][D][sensor:094]: 'neotest Display Current brightness': Sending state 81.00000 % with 0 decimals of accuracy [09:55:47][D][sensor.current_brightness:1366]: Current brightness: 81% [09:56:00][D][script.check_versions:1744]: ESPHome version: 4.2.4 [09:56:00][D][script.check_versions:1745]: TFT version: 4.2.4

 [09:56:00][D][script.check_versions:1747]: Blueprint version: [09:56:00][E][script.check_versions:1748]: Blueprint version mismatch! 

[09:56:00][D][script.check_versions:1756]: Framework: esp-idf [09:56:00][D][script.check_versions:1757]: Baud rate: 115200 [09:56:12][D][script.timer_sleep:2967]: Going to sleep from page home [09:56:12][D][script.set_brightness:2504]: brightness: 0% [09:56:12][D][nextion:464]: Got new page: 9 [09:56:12][D][display.disp1.on_page:716]: Nextion page changed [09:56:12][D][display.disp1.on_page:717]: New page: screensaver (9) [09:56:12][D][text_sensor:064]: 'neotest Current page': Sending state 'screensaver' [09:56:12][D][text_sensor:064]: 'neotest Detailed Entity': Sending state '' [09:56:12][D][script.page_changed:2110]: New page: screensaver [09:56:12][D][sensor:094]: 'neotest Display Current brightness': Sending state 0.00000 % with 0 decimals of accuracy [09:56:12][D][sensor.current_brightness:1366]: Current brightness: 0% [09:56:12][D][sensor:094]: 'neotest Page Id': Sending state 9.00000 with 0 decimals of accuracy [09:56:12][D][sensor.page_id:1381]: New page Id: 9 [09:56:12][D][sensor:094]: 'neotest Display Current brightness': Sending state 0.00000 % with 0 decimals of accuracy


The following YAML I am using:


substitutions:
  # Settings - Editable values
  device_name: "neotest" 
  wifi_ssid: "-------"
  wifi_password: "-------"
  nextion_update_url: "http://homeassistant.local:8123/local/nspanel_eu.tft"  # Optional for `esp-idf` framework
  # Add-on configuration (if needed)
  # heater_relay: "1"  # Possible values: "1" or "2"

# Customization area
##### My customization - Start #####
##### My customization - End #####

# Core and optional configurations
packages:
  remote_package:
    url: https://github.com/Blackymas/NSPanel_HA_Blueprint
    ref: main
    files:
      - nspanel_esphome.yaml # Core package
      # Optional advanced and add-on configurations
      # - advanced/esphome/nspanel_esphome_advanced.yaml
      # - nspanel_esphome_addon_climate_cool.yaml
      # - nspanel_esphome_addon_climate_heat.yaml
      # - nspanel_esphome_addon_climate_dual.yaml
    refresh: 300s
 # Optional manual IP
  
wifi:
  manual_ip:
    static_ip: 192.168.2.216
    gateway: 192.168.2.254
    subnet: 255.255.255.0

esp32:
  framework:
    type: esp-idf

**Comment by edwardtfn** [comment]
Could you please share logs which I includes a few seconds before you press the Upload TFT button the whole upload process, the restart after that and a few seconds after that restart?
Please don't reply via email with logs as that gets messed up it was quite a work to fix the logs you shared earlier.

**Comment by edwardtfn** [comment]
Hey @Pheerink, any update to this?
Are you still facing this issue or should we close this?

**Comment by Pheerink** [comment]
Yes still the same problem. My whole configuration is messed up.  I see still entities which have been deleted by removing config and create new ones.. I need to to the install HA again ?RegardsPaul Op 30 jan 2024 om 18:45 heeft Edward Firmo ***@***.***> het volgende geschreven:﻿
Hey @Pheerink, any update to this?
Are you still facing this issue or should we close this?

**Comment by edwardtfn** [comment]
There's no need to install HA again, this shoukd be sufficient to get rid of legacy entities:
1. Go to Settings --> Devices & Services --> ESPHome
2. Delete the device
3. Restart Home Assistant host
4. Go back to Settings --> Devices & Services
5. Click Add integration
6. Select ESPHome
7. Enter your panel's hostname or IP address.

**Comment by Pheerink** [comment]
Thanks for your advice but tried that dozens of times without luck. In the beginning I had a problem that a zigbee device what was in the test Ha reacting on a button pressed on the Nspanel reacted in de production HA on the same button  without configuring it in the new HA.  Does that make sense. Not to me. I am now out on holidays. Next week I will work on it again. But any advice would be appreciated. Thanks again. RegardsPaul. Op 30 jan 2024 om 20:28 heeft Edward Firmo ***@***.***> het volgende geschreven:﻿
There's no need to install HA again, this shoukd be sufficient to get rid of legacy entities:

Go to Settings --> Devices & Services --> ESPHome
Delete the device
Restart Home Assistant host
Go back to Settings --> Devices & Services
Click Add integration
Select ESPHome
Enter your panel's hostname or IP address.

**Comment by edwardtfn** [comment]
Ok. I will need mor logs when you come back.
Please take a look at Home Assistant logs (Settings > System > Logs) for anything related to this Blueprint.

**Comment by hBrinkmann62** [comment]
I have the same issue after update to 4.2.4. The display shows only the time...

![NSPanel](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/147155340/31874b61-578c-4fa0-8e4f-10dba21c8cb1)

INFO ESPHome 2023.12.9
INFO Reading configuration /config/esphome/nspanel01.yaml...
INFO Updating https://github.com/Blackymas/NSPanel_HA_Blueprint@beta
INFO Updating https://github.com/esphome/esphome.git@pull/6051/head
INFO Detected timezone 'Europe/Berlin'
INFO Starting log output from 192.168.1.140 using esphome API
INFO Successfully connected to nspanel01 @ 192.168.1.140 in 0.019s
INFO Successful handshake with nspanel01 @ 192.168.1.140 in 0.033s
[18:37:00][I][app:102]: ESPHome version 2023.12.9 compiled on Feb  2 2024, 18:18:00
[18:37:00][C][wifi:573]: WiFi:
[18:37:00][C][wifi:405]:   Local MAC: 3C:E9:0E:85:D4:A8
[18:37:00][C][wifi:410]:   SSID: [redacted]
[18:37:00][C][wifi:411]:   IP Address: 192.168.1.140
[18:37:00][C][wifi:413]:   BSSID: [redacted]
[18:37:00][C][wifi:414]:   Hostname: 'nspanel01'
[18:37:00][C][wifi:416]:   Signal strength: -51 dB ▂▄▆█
[18:37:00][C][wifi:420]:   Channel: 1
[18:37:00][C][wifi:421]:   Subnet: 255.255.255.0
[18:37:00][C][wifi:422]:   Gateway: 192.168.1.1
[18:37:00][C][wifi:423]:   DNS1: 192.168.1.1
[18:37:00][C][wifi:424]:   DNS2: 0.0.0.0
[18:37:00][C][logger:439]: Logger:
[18:37:00][C][logger:440]:   Level: DEBUG
[18:37:00][C][logger:441]:   Log Baud Rate: 0
[18:37:00][C][logger:443]:   Hardware UART: UART0
[18:37:00][C][uart.arduino_esp32:137]: UART Bus 0:
[18:37:00][C][uart.arduino_esp32:138]:   TX Pin: GPIO16
[18:37:00][C][uart.arduino_esp32:139]:   RX Pin: GPIO17
[18:37:00][C][uart.arduino_esp32:141]:   RX Buffer Size: 256
[18:37:00][C][uart.arduino_esp32:143]:   Baud Rate: 115200 baud
[18:37:00][C][uart.arduino_esp32:144]:   Data Bits: 8
[18:37:00][C][uart.arduino_esp32:145]:   Parity: NONE
[18:37:00][C][uart.arduino_esp32:146]:   Stop bits: 1
[18:37:00][C][ledc.output:164]: LEDC Output:
[18:37:00][C][ledc.output:165]:   Pin GPIO21
[18:37:00][C][ledc.output:166]:   LEDC Channel: 0
[18:37:00][C][ledc.output:167]:   PWM Frequency: 1000.0 Hz
[18:37:00][C][ledc.output:168]:   Bit depth: 16
[18:37:00][C][template.binary_sensor:028]: Template Binary Sensor 'nspanel01 Nextion display'
[18:37:00][C][template.binary_sensor:028]:   Device Class: 'connectivity'
[18:37:00][C][template.number:050]: Template Number 'nspanel01 Display Brightness'
[18:37:00][C][template.number:050]:   Unit of Measurement: '%'
[18:37:00][C][template.number:051]:   Optimistic: YES
[18:37:00][C][template.number:052]:   Update Interval: 60.0s
[18:37:00][C][template.number:050]: Template Number 'nspanel01 Display Brightness Dimdown'
[18:37:00][C][template.number:050]:   Unit of Measurement: '%'
[18:37:00][C][template.number:051]:   Optimistic: YES
[18:37:00][C][template.number:052]:   Update Interval: 60.0s
[18:37:00][C][template.number:050]: Template Number 'nspanel01 Display Brightness Sleep'
[18:37:00][C][template.number:050]:   Unit of Measurement: '%'
[18:37:00][C][template.number:051]:   Optimistic: YES
[18:37:00][C][template.number:052]:   Update Interval: 60.0s
[18:37:00][C][template.number:050]: Template Number 'nspanel01 Temperature Correction'
[18:37:00][C][template.number:050]:   Unit of Measurement: '°C'
[18:37:00][C][template.number:051]:   Optimistic: YES
[18:37:00][C][template.number:052]:   Update Interval: 60.0s
[18:37:00][C][template.number:050]: Template Number 'nspanel01 Timeout Page'
[18:37:00][C][template.number:050]:   Icon: 'mdi:timer'
[18:37:00][C][template.number:050]:   Unit of Measurement: 's'
[18:37:00][C][template.number:051]:   Optimistic: YES
[18:37:00][C][template.number:052]:   Update Interval: 60.0s
[18:37:00][C][template.number:050]: Template Number 'nspanel01 Timeout Dimming'
[18:37:00][C][template.number:050]:   Icon: 'mdi:timer'
[18:37:00][C][template.number:050]:   Unit of Measurement: 's'
[18:37:00][C][template.number:051]:   Optimistic: YES
[18:37:00][C][template.number:052]:   Update Interval: 60.0s
[18:37:00][C][template.number:050]: Template Number 'nspanel01 Timeout Sleep'
[18:37:00][C][template.number:050]:   Icon: 'mdi:timer'
[18:37:00][C][template.number:050]:   Unit of Measurement: 's'
[18:37:00][C][template.number:051]:   Optimistic: YES
[18:37:00][C][template.number:052]:   Update Interval: 60.0s
[18:37:00][C][template.select:065]: Template Select 'nspanel01 Baud rate'
[18:37:00][C][template.select:065]:   Icon: 'mdi:swap-horizontal'
[18:37:00][C][template.select:066]:   Update Interval: 60.0s
[18:37:00][C][template.select:069]:   Optimistic: YES
[18:37:00][C][template.select:070]:   Initial Option: 115200
[18:37:00][C][template.select:071]:   Restore Value: YES
[18:37:00][C][template.select:065]: Template Select 'nspanel01 Wake-up page'
[18:37:00][C][template.select:065]:   Icon: 'mdi:page-next-outline'
[18:37:00][C][template.select:066]:   Update Interval: 60.0s
[18:37:00][C][template.select:069]:   Optimistic: YES
[18:37:00][C][template.select:070]:   Initial Option: home
[18:37:00][C][template.select:071]:   Restore Value: YES
[18:37:01][C][template.select:065]: Template Select 'nspanel01 Update TFT display - Branch'
[18:37:01][C][template.select:065]:   Icon: 'mdi:file-sync'
[18:37:01][C][template.select:066]:   Update Interval: 60.0s
[18:37:01][C][template.select:069]:   Optimistic: YES
[18:37:01][C][template.select:070]:   Initial Option: main
[18:37:01][C][template.select:071]:   Restore Value: YES
[18:37:01][C][template.select:065]: Template Select 'nspanel01 Update TFT display - Model'
[18:37:01][C][template.select:065]:   Icon: 'mdi:file-sync'
[18:37:01][C][template.select:066]:   Update Interval: 60.0s
[18:37:01][C][template.select:069]:   Optimistic: YES
[18:37:01][C][template.select:070]:   Initial Option: Use nextion_update_url
[18:37:01][C][template.select:071]:   Restore Value: YES
[18:37:01][C][template.text_sensor:020]: Template Sensor 'nspanel01 Detailed Entity'
[18:37:01][C][template.text_sensor:020]:   Icon: 'mdi:tablet-dashboard'
[18:37:01][C][template.text_sensor:020]: Template Sensor 'nspanel01 Current page'
[18:37:01][C][template.text_sensor:020]:   Icon: 'mdi:tablet-dashboard'
[18:37:01][C][template.text_sensor:020]: Template Sensor 'nspanel01 Notification Label'
[18:37:01][C][template.text_sensor:020]: Template Sensor 'nspanel01 Notification Text'
[18:37:01][C][switch.gpio:068]: GPIO Switch 'nspanel01 Relay 1'
[18:37:01][C][switch.gpio:091]:   Restore Mode: restore defaults to OFF
[18:37:01][C][switch.gpio:031]:   Pin: GPIO22
[18:37:01][C][switch.gpio:068]: GPIO Switch 'nspanel01 Relay 2'
[18:37:01][C][switch.gpio:091]:   Restore Mode: restore defaults to OFF
[18:37:01][C][switch.gpio:031]:   Pin: GPIO19
[18:37:01][C][gpio.binary_sensor:015]: GPIO Binary Sensor 'nspanel01 Left Button'
[18:37:01][C][gpio.binary_sensor:016]:   Pin: GPIO14
[18:37:01][C][gpio.binary_sensor:015]: GPIO Binary Sensor 'nspanel01 Right Button'
[18:37:01][C][gpio.binary_sensor:016]:   Pin: GPIO27
[18:37:01][C][switch.gpio:068]: GPIO Switch 'nspanel01 Nextion display - Power'
[18:37:01][C][switch.gpio:091]:   Restore Mode: always ON
[18:37:01][C][switch.gpio:031]:   Pin: GPIO4
[18:37:02][C][template.switch:068]: Template Switch 'nspanel01 Notification sound'
[18:37:02][C][template.switch:091]:   Restore Mode: restore defaults to OFF
[18:37:02][C][template.switch:057]:   Optimistic: YES
[18:37:02][C][template.switch:068]: Template Switch 'nspanel01 Relay 1 Local'
[18:37:02][C][template.switch:091]:   Restore Mode: restore defaults to OFF
[18:37:02][C][template.switch:057]:   Optimistic: YES
[18:37:02][C][template.switch:068]: Template Switch 'nspanel01 Relay 2 Local'
[18:37:02][C][template.switch:091]:   Restore Mode: restore defaults to OFF
[18:37:02][C][template.switch:057]:   Optimistic: YES
[18:37:02][C][template.switch:068]: Template Switch 'nspanel01 Notification unread'
[18:37:02][C][template.switch:091]:   Restore Mode: always OFF
[18:37:02][C][template.switch:057]:   Optimistic: YES
[18:37:02][C][rtttl:029]: Rtttl
[18:37:02][C][nextion:128]: Nextion:
[18:37:02][C][nextion:129]:   Device Model:     NX4832F035_011C
[18:37:02][C][nextion:130]:   Firmware Version: 72
[18:37:02][C][nextion:131]:   Serial Number:    41B0340120ECCC32
[18:37:02][C][nextion:132]:   Flash Size:       16777216
[18:37:02][C][nextion:133]:   Wake On Touch:    YES
[18:37:02][C][nextion:134]:   Exit reparse:     YES
[18:37:02][C][restart.button:017]: Restart Button 'nspanel01 Restart'
[18:37:02][C][adc:097]: ADC Sensor 'ntc_source'
[18:37:02][C][adc:097]:   Device Class: 'voltage'
[18:37:02][C][adc:097]:   State Class: 'measurement'
[18:37:02][C][adc:097]:   Unit of Measurement: 'V'
[18:37:02][C][adc:097]:   Accuracy Decimals: 2
[18:37:02][C][adc:107]:   Pin: GPIO38
[18:37:02][C][adc:122]:  Attenuation: 11db
[18:37:02][C][adc:142]:   Update Interval: 60.0s
[18:37:02][C][resistance:010]: Resistance Sensor 'resistance_sensor'
[18:37:02][C][resistance:010]:   State Class: 'measurement'
[18:37:02][C][resistance:010]:   Unit of Measurement: 'Ω'
[18:37:02][C][resistance:010]:   Accuracy Decimals: 1
[18:37:02][C][resistance:010]:   Icon: 'mdi:flash'
[18:37:02][C][resistance:011]:   Configuration: DOWNSTREAM
[18:37:02][C][resistance:012]:   Resistor: 11200.00Ω
[18:37:02][C][resistance:013]:   Reference Voltage: 3.3V
[18:37:02][C][ntc:014]: NTC Sensor 'nspanel01 Temperature'
[18:37:02][C][ntc:014]:   Device Class: 'temperature'
[18:37:02][C][ntc:014]:   State Class: 'measurement'
[18:37:02][C][ntc:014]:   Unit of Measurement: '°C'
[18:37:02][C][ntc:014]:   Accuracy Decimals: 1
[18:37:02][C][homeassistant.time:010]: Home Assistant Time:
[18:37:02][C][homeassistant.time:011]:   Timezone: 'CET-1CEST,M3.5.0,M10.5.0/3'
[18:37:02][C][status:034]: Status Binary Sensor 'nspanel01 Status'
[18:37:02][C][status:034]:   Device Class: 'connectivity'
[18:37:03][C][mdns:115]: mDNS:
[18:37:03][C][mdns:116]:   Hostname: nspanel01
[18:37:03][C][ota:097]: Over-The-Air Updates:
[18:37:03][C][ota:098]:   Address: nspanel01.local:3232
[18:37:03][C][ota:101]:   Using Password.
[18:37:03][C][api:139]: API Server:
[18:37:03][C][api:140]:   Address: nspanel01.local:6053
[18:37:03][C][api:144]:   Using noise encryption: NO
[18:37:03][C][wifi_signal.sensor:009]: WiFi Signal 'nspanel01 RSSI'
[18:37:03][C][wifi_signal.sensor:009]:   Device Class: 'signal_strength'
[18:37:03][C][wifi_signal.sensor:009]:   State Class: 'measurement'
[18:37:03][C][wifi_signal.sensor:009]:   Unit of Measurement: 'dBm'
[18:37:03][C][wifi_signal.sensor:009]:   Accuracy Decimals: 0
[18:37:15][I][addon_upload_tft.script.report_settings:189]: TFT URL: http://192.168.1.82:8123/local/nspanel_eu.tft
[18:37:15][I][addon_upload_tft.script.report_settings:190]: Substitutions:
[18:37:15][I][addon_upload_tft.script.report_settings:191]:   nextion_update_url:        http://192.168.1.82:8123/local/nspanel_eu.tft
[18:37:15][I][addon_upload_tft.script.report_settings:192]:   nextion_update_base_url:   https://raw.githubusercontent.com/Blackymas/NSPanel_HA_Blueprint/
[18:37:15][I][addon_upload_tft.script.report_settings:193]:   upload_tft_chunk_size_max: 32768
[18:37:15][I][addon_upload_tft.script.report_settings:194]:   upload_tft_baud_rate:      921600
[18:37:30][I][script.watchdog:3115]: Wi-Fi:         Good (-51 dBm)
[18:37:30][I][script.watchdog:3135]: API:           Connected
[18:37:30][W][script.watchdog:3155]: Blueprint:     Pending
[18:37:30][I][script.watchdog:3156]: Requesting blueprint settings
[18:37:30][I][script.watchdog:3166]: ESPHome:
[18:37:30][I][script.watchdog:3167]:   Version:     4.2.4
[18:37:30][I][script.watchdog:3170]:   Framework:   arduino
[18:37:30][I][script.watchdog:3176]: UART:
[18:37:30][I][script.watchdog:3177]:   Baud rate:   115200 bps
[18:37:30][I][script.watchdog:3178]:   Queue size:  8
[18:37:30][I][script.watchdog:3182]: Nextion:
[18:37:30][I][script.watchdog:3183]:   Queue size:  2
[18:37:30][I][script.watchdog:3185]:   Is setup:    True
[18:37:30][I][script.watchdog:3192]:   Init:        True
[18:37:30][I][script.watchdog:3200]:   TFT:         4.2.4
[18:37:30][W][component:214]: Component time took a long time for an operation (0.07 s).
[18:37:30][W][component:215]: Components should block for at most 20-30ms.
[18:37:38][D][sensor:094]: 'nspanel01 RSSI': Sending state -51.00000 dBm with 0 decimals of accuracy
[18:37:49][D][sensor:094]: 'ntc_source': Sending state 1.60000 V with 2 decimals of accuracy
[18:37:49][D][resistance:039]: 'resistance_sensor' - Resistance 10541.2Ω
[18:37:49][D][sensor:094]: 'resistance_sensor': Sending state 10541.17676 Ω with 1 decimals of accuracy
[18:37:49][D][ntc:026]: 'nspanel01 Temperature' - Temperature: 23.8°C
[18:37:49][D][sensor:094]: 'nspanel01 Temperature': Sending state 24.21861 °C with 1 decimals of accuracy
[18:38:30][I][script.watchdog:3115]: Wi-Fi:         Good (-51 dBm)
[18:38:30][I][script.watchdog:3135]: API:           Connected
[18:38:30][W][script.watchdog:3155]: Blueprint:     Pending
[18:38:30][I][script.watchdog:3156]: Requesting blueprint settings
[18:38:30][I][script.watchdog:3166]: ESPHome:
[18:38:30][I][script.watchdog:3167]:   Version:     4.2.4
[18:38:30][I][script.watchdog:3170]:   Framework:   arduino
[18:38:30][I][script.watchdog:3176]: UART:
[18:38:30][I][script.watchdog:3177]:   Baud rate:   115200 bps
[18:38:30][I][script.watchdog:3178]:   Queue size:  8
[18:38:30][I][script.watchdog:3182]: Nextion:
[18:38:30][I][script.watchdog:3183]:   Queue size:  2
[18:38:30][I][script.watchdog:3185]:   Is setup:    True
[18:38:30][I][script.watchdog:3192]:   Init:        True
[18:38:30][I][script.watchdog:3200]:   TFT:         4.2.4

**Comment by Pheerink** [comment]
Is this what you need? I removed the device again and started all over. No difference!

HA log:

2024-02-08 18:59:39.489 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration sonoff which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2024-02-08 18:59:39.491 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration hacs which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2024-02-08 18:59:39.493 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration pyscript which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2024-02-08 19:00:04.052 WARNING (MainThread) [homeassistant.setup] Setup of input_text is taking over 10 seconds.
2024-02-08 19:00:04.052 WARNING (MainThread) [homeassistant.setup] Setup of schedule is taking over 10 seconds.
2024-02-08 19:00:09.358 WARNING (MainThread) [homeassistant.components.climate] Entity sonoff.sonoff_1002091763 (<class 'custom_components.sonoff.climate.XClimateNS'>) implements HVACMode(s): off, heat_cool, auto and therefore implicitly supports the turn_on/turn_off methods without setting the proper ClimateEntityFeature. Please create a bug report at https://github.com/AlexxIT/SonoffLAN/issues
2024-02-08 19:20:17.939 ERROR (MainThread) [homeassistant] Error doing job: Task exception was never retrieved
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 453, in async_run
    raise err
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 435, in async_run
    await self._async_step(log_exceptions=False)
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 487, in _async_step
    self._handle_exception(
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 512, in _handle_exception
    raise exception
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 485, in _async_step
    await getattr(self, handler)()
  File "/usr/src/homeassistant/homeassistant/helpers/trace.py", line 283, in async_wrapper
    await func(*args)
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 1096, in _async_parallel_step
    raise result
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 1088, in async_run_with_trace
    await self._async_run_script(script)
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 1100, in _async_run_script
    result = await self._async_run_long_action(
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 685, in _async_run_long_action
    return long_task.result()
           ^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 1600, in async_run
    return await asyncio.shield(run.async_run())
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 453, in async_run
    raise err
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 435, in async_run
    await self._async_step(log_exceptions=False)
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 487, in _async_step
    self._handle_exception(
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 512, in _handle_exception
    raise exception
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 485, in _async_step
    await getattr(self, handler)()
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 1076, in _async_stop_step
    raise _StopScript(stop, response)
homeassistant.helpers.script._StopScript: Stop the automation

**Comment by edwardtfn** [comment]
I cannot find anything related to the panel in the logs you shared.

> Have you tried the steps from the guide?
https://github.com/Blackymas/NSPanel_HA_Blueprint/blob/main/docs/error_initializing.md#blueprint-is-not-detected

Have you tried this?

**Comment by Pheerink** [comment]
Yes I tried that several times! I have also a Nspanel pro with nspanelmanager running. Can that interfere with it ?

**Comment by Pheerink** [comment]
It is working now. This what I did: I moved the device back to the test instance and updated everything. To my surprise it was working again. In the meantime the version number of the blueprint was changed! Then I moved it back to the Prod instance and it was working also! I'm lost but happy that it is working! 

Only one remark I have now a red vertical line on the right side of the button pages (only there). I can live with it but if there is solution I would like to know. Thanks you for all your help.
